### PR TITLE
create cart page

### PR DESCRIPTION
**Pull request by git-0r**
# Created cart page
ecom-cart

## Files to review: 
 - Cart.js
 - CartProductsList.js
 - CartOrderDetails.js
 - cartReducer.js

### CODE REVIEW

**Review by Naman-Saxena1 — APPROVED** [review]
Code syntax, variable naming and UI looks good to me for merging 👍

**Review by sri-rishi — APPROVED** [review]
Code looks good to me, couldn't find any issue, you can merge  it

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
 
     return (
-        <nav className="navbar d-flex flex-justify-evenly flex-align-center">
-            <div className="nav-section-left logo d-flex flex-justify-evenly">
-                <ion-icon name="beer-outline"></ion-icon>
-                wine & spirits
-            </div>
-            <ul className="nav-section-center d-flex flex-justify-evenly">
+        <nav className="top-navigation d-flex flex-justify-evenly flex-align-center">
+            <Link className="react-router-link" to="/">
```
Comment (nikhiljugale007) [inline src/components/Navbar.js:10]: suggestion: You can use NavLink instead of Link.

**Review by nikhiljugale007 — COMMENTED** [review]
Suggestion: try to add error handling in your API calls.
Added one suggestion, rest is good to merge.

**Pull request merged**